CHANGELOG.md

```diff
@@ -1,7 +1,71 @@
+ - Added text badges for extensions. ([#5366](https://github.com/brave/browser-laptop/issues/5366))
+ - Added selection ability for torrent files. ([#8148](https://github.com/brave/browser-laptop/issues/8148))
+ - Added temporary notice to sync prefs page regarding mobile sync and Beta label. ([#8121](https://github.com/brave/browser-laptop/issues/8121))
+ - Added a new tab page for private tabs. ([#7934](https://github.com/brave/browser-laptop/issues/7934))
+ - Added "Command+Click" to home button (opens home in a new background tab). ([#7718](https://github.com/brave/browser-laptop/issues/7718))
+ - Added origin display for downloads. ([#7468](https://github.com/brave/browser-laptop/issues/7468))
+ - Changed about:preferences#extensions link to go to the extension request category in community. ([#8252](https://github.com/brave/browser-laptop/issues/8252))
+ - Fixed Brave crash when visiting Brave Twitter page. ([#8004](https://github.com/brave/browser-laptop/issues/8004))
+ - Fixed crash caused by certain pinned tabs. ([#7187](https://github.com/brave/browser-laptop/issues/7187))
+ - Fixed opening pdf in new tab fails to load. ([#8364](https://github.com/brave/browser-laptop/issues/8364))
+ - Fixed zoom shortcut (scroll and zoom feature). ([#8438](https://github.com/brave/browser-laptop/issues/8438))
+ - Fixed bookmarks are re-synced (uploaded) on every load. ([#8408](https://github.com/brave/browser-laptop/issues/8408))
+ - Fixed messages shown on extensions page. ([#8318](https://github.com/brave/browser-laptop/issues/8318))
+ - Fixed extensions so they sort alphabetically. ([#8315](https://github.com/brave/browser-laptop/issues/8315))
+ - Fixed jumpy tab drag and drop. ([#8288](https://github.com/brave/browser-laptop/issues/8288))
+ - Fixed homepage preference isn't being respected. ([#8278](https://github.com/brave/browser-laptop/issues/8278))
+ - Fixed screen reader can't identify extension button. ([#8269](https://github.com/brave/browser-laptop/issues/8269))
+ - Fixed tabs bar size by another 2 px based on increased user feedback. ([#8263](https://github.com/brave/browser-laptop/issues/8263))
+ - Fixed memory leak when opening then closing a lot of tabs rapidly. ([#8244](https://github.com/brave/browser-laptop/issues/8244))
+ - Fixed about:brave copy to clipboard tooltip. ([#8199](https://github.com/brave/browser-laptop/issues/8199))
+ - Fixed wallet creation failure. ([#8195](https://github.com/brave/browser-laptop/issues/8195))
+ - Fixed invisible pinned tab after upgrade. ([#8190](https://github.com/brave/browser-laptop/issues/8190))
+ - Fixed URL bar shape consistency when payments are enabled/disabled. ([#8170](https://github.com/brave/browser-laptop/issues/8170))
+ - Fixed creating a bookmark by dragging the Url bar icon. ([#8151](https://github.com/brave/browser-laptop/issues/8151))
+ - Fixed save .torrent file action. ([#8146](https://github.com/brave/browser-laptop/issues/8146))
```

Comment:
this one shouldn't be in here since .torrent was introduced this release also (with https://github.com/brave/browser-laptop/issues/6671)